### DISCUSSION TITLE
add labels to filename_keys?

### DISCUSSION THREAD

**Discussion by caniyabanci76** [audioscavenger/save-image-extended-comfyui#12] [Ideas]
would it be possible to add manual strings to the keys list?

so say for filename keys , i wanted `seed`, `width`, `height`.  Currently,  we would get:  `1234567890_1024_1024_0001.png`

But I would like to prefix those values with a label, so the output would be for example: `seed=1234567890_w=1024_h=1024_0001.png`

**Comment by audioscavenger** [comment]
yes that's possible, i will add just yet another option to enabled those. now, how do you deal with multiple same widgets like 1.seed and 13.width, do you want just the names anyway? i wouldd say so, but let me know

**Comment by caniyabanci76** [comment]
just the names would be fine thanks

**Comment by audioscavenger** [comment]
release 2.75
